### PR TITLE
Cleanup/check ups

### PR DESCRIPTION
**Pull request by RincewindsHat**
* clang-format
 * copyright update

### CODE REVIEW

File: plugins/check_ups.c [hunk 1]
```diff
@@ -233,24 +230,16 @@ int main(int argc, char **argv) {
...
 				result = max_state(result, STATE_WARNING);
 			}
 			xasprintf(&data, "%s",
-					  perfdata("voltage", (long)(1000 * ups_utility_voltage),
-							   "mV", config.check_warn,
-							   (long)(1000 * config.warning_value),
-							   config.check_crit,
-							   (long)(1000 * config.critical_value), true, 0,
-							   false, 0));
+					  perfdata("voltage", (long)(1000 * ups_utility_voltage), "mV", config.check_warn, (long)(1000 * config.warning_value), config.check_crit,
```
Comment (sni) [inline plugins/check_ups.c:239]: i don't think this is better than the current format.

**Comment by RincewindsHat** [comment]
@sni: Play around a little bit with the options. What do you think about this one?

File: plugins/check_ups.c [hunk 2]
```diff
@@ -233,24 +230,26 @@ int main(int argc, char **argv) {
...
-			} else if (config.check_warn &&
-					   ups_utility_deviation >= config.warning_value) {
+			} else if (config.check_warn && ups_utility_deviation >= config.warning_value) {
 				result = max_state(result, STATE_WARNING);
 			}
-			xasprintf(&data, "%s",
-					  perfdata("voltage", (long)(1000 * ups_utility_voltage),
-							   "mV", config.check_warn,
+			xasprintf(&data,
+					  "%s",
```
Comment (sni) [inline plugins/check_ups.c:239]: looks better, except these lines with the format on a single line

File: plugins/check_ups.c [hunk 3]
```diff
@@ -706,33 +723,42 @@ void print_help(void) {
...
 	printf("\n");
-	printf("%s\n", _("This plugin attempts to determine the status of a UPS "
-					 "(Uninterruptible Power"));
-	printf("%s\n", _("Supply) on a local or remote host. If the UPS is online "
-					 "or calibrating, the"));
-	printf("%s\n", _("plugin will return an OK state. If the battery is on it "
-					 "will return a WARNING"));
-	printf("%s\n", _("state. If the UPS is off or has a low battery the plugin "
-					 "will return a CRITICAL"));
+	printf("%s\n",
```
Comment (sni) [inline plugins/check_ups.c:726]: and those...

**Review by sni — COMMENTED** [review]
small complaints, but i am willing to drop them for the sake of a consistent format.

File: plugins/check_ups.c (hunk 2 above)
Comment (RincewindsHat) [reply plugins/check_ups.c:239]: hm, i will play a bit more with the binpack parameters

Comment (RincewindsHat) [reply plugins/check_ups.c]: so, changed `BinPack`. Arguments are again more compressed. Maybe a little bit too much, but to change that the line length should be reduced I guess.
@sni: How do you like it?

**Review by sni — APPROVED** [review]

**Comment by sni** [comment]
i'd say, so far, this is the best variant. Still not perfect...

**Pull request merged**